### ISSUE TITLE
m4/ax_cxx_compile_stdcxx.m4 missing in stk-5.0.0.tar.gz

### ISSUE THREAD

**Issue by ryandesign** [thestk/stk#139]
Attempting to `autoreconf` after extracting stk-5.0.0.tar.gz fails:

```
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: /opt/local/bin/aclocal 
aclocal: error: configure.ac:11: file 'm4/ax_cxx_compile_stdcxx.m4' does not exist
autoreconf: error: /opt/local/bin/aclocal failed with exit status: 1
```

**Comment by garyscavone** [comment]
Yes, I inadvertently deleted the m4 directory in that distribution. I'll create a new one in the next day or so. In the mean time, you can just checkout from github.

**Comment by garyscavone** [comment]
This is fixed in the 5.0.1 release distribution at CCRMA.
